packages/core/content-releases/admin/src/pages/tests/Releases.test.tsx

```diff
@@ -0,0 +1,116 @@
+import { lightTheme, ThemeProvider } from '@strapi/design-system';
+import { render as renderRTL, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from 'react-intl';
+
+import { ReleasesPage } from '../Releases';
+
+const render = () => ({
+  ...renderRTL(<ReleasesPage />, {
+    wrapper: ({ children }) => (
+      <ThemeProvider theme={lightTheme}>
+        <IntlProvider locale="en" messages={{}} defaultLocale="en">
+          {children}
+        </IntlProvider>
+      </ThemeProvider>
+    ),
+  }),
+  user: userEvent.setup(),
+});
+
+describe('Releases home page', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders the Releases title as an heading', async () => {
+    const { getByRole } = render();
+
+    const pageTitle = getByRole('heading', { level: 1 });
+    expect(pageTitle).toHaveTextContent('Releases');
+  });
+
+  it('shows a subtitle with the numnber of releases', async () => {
+    const { getByText } = render();
+
+    // if there are 0 releases
+    expect(getByText('No releases')).toBeInTheDocument();
+  });
+
+  it('shows a button to create a new release', async () => {
+    const { getByRole } = render();
+
+    const newReleaseButton = getByRole('button', { name: 'New release' });
+
+    expect(newReleaseButton).toBeInTheDocument();
+  });
+
+  it('shows a dialog when clicking on the "New release" button', async () => {
```

Comment:
General code formatting comment: here you insert blank lines between each line of code, in my opinion that doesn't make things more readable. Instead try to think, without looking at the code, what are the different steps I need for this step? 1 step should be 1 code block, with blank lines only to separate them

Reply:
I remember reading about a pattern a long time ago that helped me organize my tests called "Arrange, Act, Assert". Not sure the exact article but I think there are a lot out there that explain the idea, just thought I'd share :)